components/registration/registration.html

```diff
@@ -89,7 +89,8 @@
         <!-- registration buttons begin -->
         <div ng-if="!editingDisabled && registrationMode === 'PROFILE'" class="vertical-spacing btn-group">
             <button type="button" class="btn btn-primary" ng-disabled="selectedOrgUnit.closedStatus" ng-click="registerEntity(null)">{{'save'| translate}}</button>
-            <button type="button" class="btn btn-default small-horizontal-spacing" ng-disabled="selectedOrgUnit.closedStatus" ng-click="cancelRegistrationWarning(cancel)">{{'cancel'| translate}}</button>
+            <button type="button" class="btn btn-default small-horizontal-spacing" ng-disabled="selectedOrgUnit.closedStatus" ng-click="cancelRegistrationWarning(cancel)" ng-if="!isInDashboard">{{'cancel'| translate}}</button>
```

Comment:
I think the parameter isInDashboard might be redundant here. The $scope.isInDashboard should be available within the function without you passing it in.